src/scripts.js

```diff
@@ -173,12 +177,41 @@ function populateSearchResults(searchWord) {
     allFoundRecipes = allFoundRecipes.concat(recipe.searchByIngredient(searchWord, allInstantiatedRecipes))
   }
 
-  console.log(allFoundRecipes);
   if (allFoundRecipes.length === 0) {
     featuredRecipeSection.remove();
-    allRecipesSection.insertAdjacentHTML('afterbegin', `<h1>We're Sorry there are no recipes that match your search result</h1>`)
+    allRecipesSection.innerHTML = '';
+    allRecipesSection.insertAdjacentHTML('afterbegin', `<h1>We're Sorry! There are no recipes that match your search result.</h1>`)
   } else {
   featuredRecipeSection.remove();
   populateRecipeCards(allFoundRecipes, allRecipesSection)
   }
 }
+
+function displayEntireRecipe(event) {
+  selectedRecipe.innerHTML = '';
+  let recipeCardId = Number(event.target.id);
+  let foundRecipe = allInstantiatedRecipes.find(recipe => recipe.id === recipeCardId);
+  allRecipesSection.innerHTML = '';
+  selectedRecipe.classList.remove('hidden');
+  featuredRecipeSection.remove();
+
+  selectedRecipe.insertAdjacentHTML('afterbegin', `<img src=${foundRecipe.image} alt="" class="featured-recipe-photo">
```

Comment:
After so much driver/navigator via Tuple, I can hear your voice in my head while reading this code.